docs/sources/explore/logs-integration.md

```diff
@@ -0,0 +1,120 @@
+| notice                      | info          | green       |
+| dbug                        | debug         | blue        |
+| debug                       | debug         | blue        |
+| trace                       | trace         | light blue  |
+| *                           | unknown       | grey        |
+
+### Visualization options
+
+You can customize how logs are displayed and select which columns are shown.
+
+#### Time
+
+Shows or hides the time column. This is the timestamp associated with the log line as reported from the data source.
+
+#### Unique labels
+
+Shows or hides the unique labels column that includes only non-common labels. All common labels are displayed above.
+
+#### Wrap lines
+
+Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.
+
+#### Deduping
+
+Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
+
+- **Exact -** Exact matches are done on the whole line except for date fields.
+- **Numbers -** Matches on the line after stripping out numbers such as durations, IP addresses, and so on.
+- **Signature -** The most aggressive deduping, this strips all letters and numbers and matches on the remaining whitespace and punctuation.
+
+#### Flip results order
+
+You can change the order of received logs from the default descending order (newest first) to ascending order (oldest first).
+
+### Labels and detected fields
+
+Each log row has an extendable area with its labels and detected fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected labels. Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.
+
+### Toggle detected fields
+
+> **Note:** This feature is only available in Grafana 7.2+.
+
+If your logs are structured in `json` or `logfmt`, then you can show or hide detected fields. Expand a log line and then click the eye icon to show or hide fields.
+
+{{< docs-imagebox img="/img/docs/explore/parsed-fields-7-2.gif" max-width="800px" caption="Toggling detected fields in Explore" >}}
+
+### Loki-specific features
+
+As mentioned, one of the log integrations is for the new open source log aggregation system from Grafana Labs - [Loki](https://github.com/grafana/loki). Loki is designed to be very cost effective, as it does not index the contents of the logs, but rather a set of labels for each log stream. The logs from Loki are queried in a similar way to querying with label selectors in Prometheus. It uses labels to group log streams which can be made to match up with your Prometheus labels. For more information about Grafana Loki, refer to [Grafana Loki](https://github.com/grafana/loki) or the Grafana Labs hosted variant: [Grafana Cloud Logs](https://grafana.com/loki).
```

Comment:
I suspect that these links predate the Loki docs being integrated into the website.

@cyriltovena, are there better places for the links in this paragraph to point to?

Reply:
We can revisit this for v8.0?

Reply:
https://grafana.com/oss/loki/ I think the best link we have.